.github/workflows/dhis2-verify-app.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     call-workflow-e2e-prod:
         if: "!contains(github.event.head_commit.message, '[skip ci]')"
         needs: [build, lint, test]
-        uses: dhis2/data-visualizer-app/.github/workflows/e2e-prod.yml@dev
+        uses: ./.github/workflows/e2e-prod.yml
```

Comment:
Are these workflows different now Could we port the changes to the "shared" copy in DV. Btw, the DV location is only temporary until we decide how to manage our shared workflows (e.g., the workflows repo)

Reply:
Good question. Yes there are some minor differences between what we have in DV and what we have here now. And I agree it'd be good to use the same workflow for LL and DV, or ideally for all 4 of our apps.

I'll bare this in mind when I upgrade DV.